backend/supabase/group_invites/create.sql

```diff
@@ -0,0 +1,45 @@
+drop table group_invites;
+
+create table if not exists
+  group_invites (
+    id text not null primary key default random_alphanumeric (12),
```

Comment:
I think Sinclair did this for the id before:

```
create table if not exists
  market_ads (
    id text not null primary key default uuid_generate_v4 (),
 ```

Reply:
Yeah but this ID is hella long, I think it would be ugly in a share link